components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -23,6 +24,15 @@ export const getSchoolLocationText = (hasOnlyOnline, locations) => {
   return locationText;
 };
 
+// eslint-disable-next-line react/prop-types
+const LabelWithScreenReader = ({ isActive, label }) => (
+  <>
+    <ScreenReaderOnly>{isActive ? 'Has ' : 'Does not have '}</ScreenReaderOnly>
+    {label}
+    <ScreenReaderOnly>availability</ScreenReaderOnly>
```

Comment:
I think you can drop this line.

Reply:
Not sure which line you're talking about, it's not highlighted for me.

Reply:
Of the 3 lines:
```
<ScreenReaderOnly>{isActive ? 'Has ' : 'Does not have '}</ScreenReaderOnly>
  {label}
<ScreenReaderOnly>availability</ScreenReaderOnly>
```

I only want the first 2 within the fragment.